### PR TITLE
feat: prepare seperate release for package constants

### PR DESCRIPTION
**Pull request by jw-foss**
* Add necessary metadata to constants/package.json.
* Add tsconfig for local typing.
* Add clean-package.json for package.json cleanups.

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by btea — APPROVED** [review]

File: packages/constants/package.json [hunk 1]
```diff
@@ -1,6 +1,46 @@
 {
   "name": "@element-plus/constants",
   "private": true,
+  "homepage": "https://github.com/element-plus/element-plus#readme",
```
Comment (btea) [inline packages/constants/package.json:4]: It seems there is still a version missing.

Comment (jw-foss) [reply packages/constants/package.json:4]: Did you mean the "version" field? Well, that field will be added once everything is ready, so to keep the process intact I don't want to add those fields related to the publishing stuff.

Comment (btea) [reply packages/constants/package.json:4]: Yes. OK, got it.

Comment (YunYouJun) [reply packages/constants/package.json:4]: We can create a release script to update all version later.

**Pull request merged**